app/frontend/styles/components/vacancy_form.scss

```diff
@@ -0,0 +1,54 @@
+@media (min-width: 40.0625em) {
```

Comment:
Why can't we use a link instead of styling a button like a link?

Reply:
It needs to submit a form. The 'submit' buttons which come out of the box of simple_form and govuk form builder are buttons. Alternatively I could have tried to submit the form using a link + JavaScript, but it looked like that would have been (a) complicated and (b) too far from convention to be able to get much help online or in docs.

I suspect that it might be good for accessibility reasons to conform to the expected form layour, i.e. a form containing <input type="submit" .../>

Reply:
100% correct although you can use button and input of type submit these days. a link is very inaccesible and requires JS behaviour. you should be able to submit the form using the keyboard return key (as well as clicking obviously)